app/es_api/utils/queries.py

```diff
@@ -136,6 +137,35 @@ def more_like_this(doc_id):
     return response
 
 
+def spotlight_courses():
```

Comment:
Just wondering -- is there a particular reason that we are going to the search engine for these courses, when we know the specific keys/ids?  In the case where we have the actual identifiers I would think it would be better to query the XIS, yes?

Reply:
So far all access to records from the XDS component has been through the search engine so I assumed that was the pattern we were following. I agree with going directly to the XIS in this instance.